src/common/utils.tsx

```diff
@@ -15,3 +15,22 @@ export function readFileAsJson<T>(file: File): Promise<T> {
     reader.readAsText(file);
   });
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getAcceptedFormatValue = (schema: any): string => {
```

Comment:
Can remove this if we are not doing the json schema method

Reply:
yup ok